erpnext/hr/doctype/attendance_status/attendance_status.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe import _, bold
+from frappe.utils import get_link_to_form
+from frappe.model.document import Document
+
+class AttendanceStatus(Document):
+	def validate(self):
+		if self.is_present and self.is_leave and self.is_half_day:
+			frappe.throw(_("Attendance status can be either Present, leave or Half day"))
```

Comment:
There can be different possibilities too, like "present" and "leave" can be checked together but not "half day" and so on. Can you take that into consideration too?

```suggestion
			frappe.throw(_("Attendance status can either be Present, Leave or Half day."))
```